docs/sources/writing-guide/topic-types/tutorial/index.md

```diff
@@ -96,6 +96,58 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
 
 If getting access to the tutorial data is complex, include the instructions in the steps of the tutorial. If getting access to the data is straightforward, include it in the "Before you begin" section.
 
+## Publish your tutorial
+
+Your tutorial source is stored in your project repo in a `tutorials` folder and mounted to the tutorials repo so that it can be displayed on the [Tutorials](/tutorials) page. The source is stored in your project repo to make it easy for team members to review and edit the content. 
+
+Follow the steps below to hide the tutorial from your project's table of contents and to display it on the Tutorials page. 
+
+### Hide your tutorial from your Table of Contents
+
+Before completing these steps, you must create a `tutorials` directory under `sources` and add your tutorial into its own subdirectory as described in the [Write a tutorial topic](#write-a-tutorial-topic) section.
+
+- Hide your tutorial from your documentation table of contents by integrating the following to the `$.cascade` field in the YAML metadata of your `sources/_index.md` file. 
```

Comment:
You could also make this a multi-step task like so:

1. Open `sources/_index.md`.
1. Add the following code to the `$.cascade` field.

[code here...]